packages/loopring_v3/contracts/impl/libexchange/ExchangeBlocks.sol

```diff
@@ -82,52 +82,88 @@ library ExchangeBlocks
         );
     }
 
-    function verifyBlock(
+    function verifyBlocks(
         ExchangeData.State storage S,
-        uint blockIdx,
-        uint256[8] memory proof
+        uint[] memory blockIndices,
```

Comment:
should we check the max length of blockIndies?

Reply:
There is no hard limit (except for the Ethereum gas limit/block). There's also a good chance verifying a proof will get even cheaper in Istanbul so it's up to the operator to decide how many proofs can be successfully verified.